templates/actors/group-sheet.hbs

```diff
@@ -0,0 +1,93 @@
+                            <i class="fas fa-cog"></i>
+                        </a>
+                    </h4>
+                    <div class="attribute-value">
+                        <span>{{movement.primary}}</span>
+                    </div>
+                    <footer class="attribute-footer">
+                        <span>{{movement.secondary}}</span>
+                    </footer>
+                </li>
+            </ul>
+        </section>
+    </header>
+
+    <nav class="sheet-navigation tabs" data-group="primary">
+        <a class="item active" data-tab="members">{{localize 'DND5E.GroupMembers'}}</a>
+        <a class="item" data-tab="inventory">{{localize 'DND5E.GroupInventory'}}</a>
+        <a class="item" data-tab="description">{{localize 'DND5E.Description'}}</a>
+    </nav>
+
+    <section class="sheet-body">
+        <div class="tab members" data-group="primary" data-tab="members">
+            {{#each sections as |section id|}}
+            <section class="group-members directory" data-section="{{id}}">
+                <header class="directory-header flexrow">
+                    <h3 class="name">{{section.label}}</h3>
+                    <span class="hp">Hit Points</span>
+                    <span class="controls">Controls</span>
+                </header>
+                <ol class="section-members directory-list">
+                    {{#each section.members as |member|}}
+                    <li class="group-member directory-item flexrow" data-actor-id="{{member.id}}">
+                        <div class="name flexrow">
+                            <img class="thumbnail" src="{{member.img}}">
+                            <h4 class="document-name"><a class="member-link">{{member.name}}</a></h4>
+                        </div>
+                        <div class="hp flexrow">
+                            <div class="bar-container">
+                                <div class="bar" style="width:{{member.hp.pct}}%; background:{{member.hp.color}}; border-color:{{member.hp.border}}"></div>
+                            </div>
+                            {{#if displayHPValues}}
+                            <span class="current">{{member.hp.current}}</span>
+                            <span class="sep">/</span>
+                            <span class="max">{{member.hp.max}}</span>
+                            {{/if}}
+                        </div>
+                        <div class="controls">
+                            <a class="action-button" data-action="removeMember"><i class="fa-solid fa-close"></i></a>
+                        </div>
+                    </li>
+                    {{/each}}
+                </ol>
+            </section>
+            {{/each}}
+        </div>
+
+        <div class="tab inventory" data-group="primary" data-tab="inventory">
+            {{> "dnd5e.actor-inventory" sections=inventory}}
```

Comment:
For some reason the inventory isn't scrollable if the number of items extends beyond the bottom of the sheet.